### DISCUSSION TITLE
Discord UDP WebRTC not working

### DISCUSSION THREAD

**Discussion by barbuslex** [shadowsocks/shadowsocks-org#203] [Q&A]
Hi,

Here my Shadowsock server config : 
```
{
    "server": "0.0.0.0",
    "mode":"tcp_and_udp",
    "server_port": 8080,
    "password": "password",
    "method": "chacha20-ietf-poly1305"
}
```

Here my client config : 
![image](https://user-images.githubusercontent.com/113057/182629841-0585e958-6f21-46c2-9e54-91c6ba1f972d.png)

Here my Firefox proxy settings : 
![image](https://user-images.githubusercontent.com/113057/182629957-4a88bfca-df53-4b3c-b5dc-fa4cbd2d046a.png)

All web are running perfectly but when i go on discord.com when i join a channel i have "ICE checking" failed in loop...
![image](https://user-images.githubusercontent.com/113057/182630333-66ffc83f-987a-46c7-bafd-e4c8470c9a59.png)

Discord use UDP port for WebRTC. Have you an idea to make it work ?

Thanks

**Comment by lzm0** [comment]
If you are using the web version of Discord. There's a Chrome plugin https://chrome.google.com/webstore/detail/webrtc-network-limiter/npeicpdbkakmehahjeeohfdhnlpdklia that forces WebRTC sessions to use proxies.

**Reply by barbuslex** [reply]
Thanks you for your reply i have see it but the proxy option is grayed. (Need beta version of google chrome). And i have try to find it on firefox i don't found anything :-S
